database/query/profiles.sql

```diff
@@ -28,7 +28,7 @@ INSERT INTO entity_profiles (
     $1,
     $2,
     sqlc.arg(contextual_rules)::jsonb,
-    FALSE
+    TRUE
```

Comment:
At this point, any new or updated profiles will have rules in the new rule instance table, and thus can be considered migrated.